Turner.Infrastructure.Mediator.EF/Configuration/SimpleInjectorMediatorConfiguration.cs

```diff
@@ -0,0 +1,15 @@
+﻿using SimpleInjector;
+using System.Reflection;
+using Turner.Infrastructure.Mediator.Decorators;
+
+namespace Turner.Infrastructure.Mediator.Configuration.EF
```

Comment:
I would suggest suggest renaming this from `EF` to `EntityFramework` or `EntityFrameworkCore`.  (I may be in the minority on this one, I'm not sure, but I generally avoid abbreviations in package names).

Reply:
You are right, I just rushed it and didn't give anything much thought besides moving the files to other projects.  Went with `EntityFrameworkCore` since that is the Microsoft namespace name.